content/help/ops/traffic-management/proxy-cmd/index.md

```diff
@@ -278,6 +278,70 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. When envoy use the `serviceName` field as a key that have got the list of Endpoints and then will proxy the request to one of them.
+
+    {{< text bash json >}}
+    $ istioctl proxy-config endpoints productpage-v1-6c886ff494-7vxhs --cluster="outbound|9080||reviews.default.svc.cluster.local" -o json
```

Comment:
--cluster outbound|9080||reviews.default.svc.cluster.local

Reply:
and could you use the summary here not `-o json`

Reply:
Done.